packages/augur-ui/src/modules/modal/containers/modal-universe-selector.ts

```diff
@@ -0,0 +1,53 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { ModalUniverseSelector } from 'modules/modal/components/modal-universe-selector';
+import { closeModal } from 'modules/modal/actions/close-modal';
+import { AppState } from 'store';
+import { ThunkDispatch } from 'redux-thunk';
+import { Action } from 'redux';
+
+const mapStateToProps = (state: AppState) => {
+  const universe = state.universe;
+  return {
+    modal: state.modal,
+    universeDetails: {
+      address: universe.address,
+      creationTimestamp: universe.creationTimestamp,
+      outcomeName: universe.outcomeName,
+      usersRep: universe.usersRep,
+      totalRepSupply: universe.totalRepSupply,
+      totalOpenInterest: universe.totalOpenInterest,
+      numberOfMarkets: universe.numberOfMarkets,
+      children: universe.children,
```

Comment:
looks like you can just return `universe` no reason to build an object that has all the same properties.

Reply:
Yep, was planning to changes this in my next PR, actually.

Reply:
I'll push my latest changes to this PR shortly.